boards/arm/nucleo_g071rb/support/openocd.cfg

```diff
@@ -0,0 +1,23 @@
+# This is an ST NUCLEO-G071RB board with single STM32G071RB chip.
+# http://www.st.com/en/evaluation-tools/nucleo-g071rb.html
+source [find interface/stlink.cfg]
+
+transport select hla_swd
+
+set WORKAREASIZE 0x2000
+
+source [find target/stm32g0.cfg]
```

Comment:
Is that working?

Reply:
Question stands

Reply:
uart:~$ flash erase 0x1E000 0x1000
uart:~$ flash write 0x1E000 00 01 02 03 04 05 06 07
Reading back written bytes:
00 01 02 03 | 04 05 06 07
uart:~$ flash read 0x1E000 10
00 01 02 03 | 04 05 06 07
ff ff